dobject.py

```diff
@@ -498,7 +498,7 @@ def _highscore_cb(self, val, score):
         for L in self._listeners:
             L(val)
 
-class AddOnlySet:
+class AddOnlySet(ListSet):
```

Comment:
Was there a reason for this change to inherit from ListSet?  It seems like a good idea, but this probably makes all of the copying of operator methods (like `self.__and__`) unnecessary.